Tasks/JenkinsDownloadArtifactsV1/make.json

```diff
@@ -1,10 +1,2 @@
 {
-    "rm": [
```

Comment:
This thing is deleting vsts-task-lib from node_modules, when you run build task from the root folder. 
Then all tests fails, because can't find vsts-task-lib. 

I decided to remove it, everything is working fine.
Tests are ok, smoke tested with Jenkins in docker and local pipeline, it is working.

Any thoughts?